### PR TITLE
refactor(pairing): move wallet pairing via QR code

### PR DESCRIPTION
**Pull request by bchrisarriola**
Tested that paring a wallet via QR code works. Will work on 2FA flow next.

### CODE REVIEW

File: Blockchain/Authentication/AuthenticationCoordinator.swift [hunk 1]
```diff
@@ -280,6 +335,7 @@ import Foundation
             topMostViewController?.present(pinViewController, animated: true) { [weak self] in
                 guard let strongSelf = self else { return }
 
+                // CA: Can both of these alerts be moved elsewhere?
```
Comment (woudini) [inline Blockchain/Authentication/AuthenticationCoordinator.swift]: What does `CA:` mean?

Comment (bchrisarriola) [reply Blockchain/Authentication/AuthenticationCoordinator.swift]: "CA" are my initials. I did that so it's clear who wrote the comment (but apparently it's not).

@achtenhagen @woudini what do you guys think about having a convention like this so when you're reading through the code you know who to talk to if you have a question about a specific comment (another alternative is to just use our first name)? Using `git blame` is another option but this is easier

Comment (woudini) [reply Blockchain/Authentication/AuthenticationCoordinator.swift]: I'd rather not use it. In Xcode you can `control` + click and go to `Show Last Change for Line`, which shows the author.

Comment (bchrisarriola) [reply Blockchain/Authentication/AuthenticationCoordinator.swift]: Ah, good to know. Let me update

**Pull request merged**